examples/fl-dp-sa/fl_dp_sa/task.py

```diff
@@ -13,10 +13,10 @@
 
 DEVICE = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
 
+fds = None  # Cache FederatedDataset
```

Comment:
(forgot to add this comment in my review before finalising it)

This `fds` isn't used. You'll need to make use of it similarly to how other `quickstart` examples or migrated examples in general use it. See this: https://github.com/adap/flower/blob/486cac3c5e68f717ad023e87e01bbb87c115c37d/examples/quickstart-pytorch/pytorchexample/task.py#L51